docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -110,6 +110,18 @@ This change affects almost all components where you're using the `component` pro
   />
   ```
 
+### RootRef
+
+- This component was removed.
+  You can get a reference to the underlying DOM node of our components via `ref` prop.
+
+  ```diff
+  -<RootRef rootRef={ref}>
+  -  <Button />
+  +<Button ref={ref} />
+  -</RootRef>
```

Comment:
I think that it would make it less confusion, I also believe it's how git show such diff:

```suggestion
  -<RootRef rootRef={ref}>
  -  <Button />
  -</RootRef>
  +<Button ref={ref} />
```